src/components/Header.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch, faShoppingCart } from '@fortawesome/free-solid-svg-icons'
+
+function Header() {
+    return (
+        <header className="wz-header">
+            <input type="checkbox" id="nav-check"></input>
+            <div className="nav-header">
+                <div className="nav-title">Furnituria</div>
+            </div>
+            <div className="nav-btn">
+                <label htmlFor="nav-check">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </label>
+            </div>
+            <div className="nav-links">
+                <a href="/#">
```

Comment:
Creo que este `<a>` valdria la pena irlo modificando para que sea un input y poder realizar busquedas desde aquí, o igual si tienes contemplado hacerlo de alguna otra forma no hay problema.

Reply:
I was thinking the same, too.
Adding an input for making the searching directly on the navbar.

Reply:
Great, you can do it later. But it's good that you are already thinking about whats next